incubator/cni-genie/templates/daemonset.yaml

```diff
@@ -0,0 +1,153 @@
+# Install CNI-Genie plugin on each slave node.
+kind: DaemonSet
+apiVersion: extensions/v1beta1
```

Comment:
Can you please use `apps/v1` for the `apiVersion`. The extensions one is deprecated and soon k8s will stop accepting them. This applies to all the daemonsets.